packages/datadog-plugin-redis/src/index.js

```diff
@@ -9,7 +9,7 @@ class RedisPlugin extends CachePlugin {
   static get system () { return 'redis' }
 
   start ({ db, command, args, connectionOptions = {}, connectionName }) {
-    if (!this.config.filter(command)) return this.skip()
+    if (!this.config.filter(command.toUpperCase())) return this.skip()
```

Comment:
This is uppercased in multiple places.

Reply:
Well, this particular string value is only capitalized once. If not that's a bug.

The command string that the user supplies via configuration at startup time needs to be normalized and stored (happens in `normalizeConfig()`). Then when the command is executed at runtime it needs to be normalized as well (here in `start()`).

Reply:
It's uppercased in `formatCommand` as well.

Reply:
Good catch, this has been fixed.